modules/ROOT/pages/troubleshooting.adoc

```diff
@@ -0,0 +1,86 @@
+
+Notice how in the following example the Backend services quota is set to '5', and we're changing it to '50'. For the K8ssandra deployments (Stargate, cass-operator, Reaper, Medusa, and so on), actually `10` might be a sufficient quota.
+
+image::gcp-quota-example2.png[GCP Backend Quota]
+
+[#containerspending]
+== Containers in pending state 
+
+If your pods are stuck in Pending state after installation, or your cloud provider is warning you about Unschedulable Pods, there are a few ways to work through this.
+
+* If some of your pods start, but others like `pulsar-adminconsole` and `pulsar-grafana` are left in an unschedulable state, you might need to add CPUs to your existing nodes, or add an additional node pool. Luna Streaming requires more resources than Pulsar. 
+
+* To examine a specific pod, use `kubectl describe`. For example, if your `pulsar-bookkeeper-0` pod is not scheduling, use `kubectl describe pods/pulsar-bookkeeper-0` to view detailed output on the pod's state, dependencies, and events. 
+
+* To view log output for a specific pod, use `kubectl logs`. Continuing with the `pulsar-bookkeeper-0` example, to get logs for this pod use `kubectl logs pulsar-bookkeeper-0`. If this pod spans multiple containers, add the flag `--all-containers=true`. 
+
+[#bastionpod]
+== Bastion Pod
+
+The bastion pod deploys with Luna Streaming and allows a user to execute administrative commands in a bash shell of the cluster. This can be a handy troubleshooting tool for testing the health of your cluster. 
+
+=== Set up bastion pod
+
+To use the bastion pod after installation, access the shell of the bastion pod (here we are using namespace `default`):
+
+----
+kubectl -n default exec $(kubectl get pods --namespace default -l "app=pulsar,component=bastion" -o jsonpath="{.items[0].metadata.name}") -it -- bash
+----
+
+This sets you up in a bash shell that looks like this: 
+
+----
+pulsar@pulsar-bastion-78d4cd74ff-ctvvh:/pulsar$
+----
+
+=== Test bastion pod
+
+Ensure your bastion pod sees your cluster with `pulsar admin namespaces list public`. It should return a list of your public namespaces:
+
+----
+"public/default"
+"public/functions"
+----
+
+If you get `bash: admin: command not found`, try moving into the `bin` directory with `cd bin`. You may need to append your command with `./`, for example `./pulsar-admin get pods`.
+If the shell finds no resources, you may not have any public namespaces. Create a public namespace with `pulsar-admin namespaces create public/default`. 
+
+=== Publish a message
+
+To test your Pulsar cluster with the bastion pod, produce a message with `pulsar-client` through the bastion pod shell: 
+
+`pulsar-client produce my-test-topic --messages "hello-pulsar"`
+
+You should receive a confirmation the message was produced:
+
+`00:16:37.970 [main] INFO  org.apache.pulsar.client.cli.PulsarClientTool - 1 messages successfully produced`
+
+This means your Pulsar cluster is functional. 
```

Comment:
Because this is troubleshooting, what would happen if it doesn't work correctly?